### ISSUE TITLE
Reading tensorflow code

### ISSUE THREAD

**Issue by alephman** [samjabrahams/tensorflow-white-pages-notes#2]
Hi Sam,
This notes is really great, do you have any plan to read tensorflow source code with this whitepaper?

**Comment by samjabrahams** [comment]
Hey @alephman - would you mind clarifying exactly what your asking? I'm exactly sure how to interpret this question. Is it about whether I'm going to add more specific source code information into this repository?

**Comment by alephman** [comment]
@samjabrahams  Yeah, the white paper only describes the main idea.   If we know the implementation of knowledge, it could be great! The tensorflow source is becoming complex:  python as the front end, c++ as the core platform, distribution training...

**Comment by samjabrahams** [comment]
Got it. Well, I've been thinking about writing a more comprehensive "tour" of the TensorFlow code base (as much as to solidify my own understanding as anything else), but I wasn't originally planning on putting it here. That said, it may make more sense to put it in this repository, as this one already has a decent online presence.

So to answer your question: I think yes! It'll take a fair amount of time, however.

**Comment by alephman** [comment]
Sounds great!

**Comment by larissa95** [comment]
Hi, have you completed the "tour"? If so, where have you put it?

**Comment by samjabrahams** [comment]
I've definitely done a lot of digging/learning, though I can't say I've "completed" it yet. It may make more sense to do section-by-section breakdowns of the codebase instead of attempting one large swath at once.

**Comment by larissa95** [comment]
If you start doing so, let me know, it would be nice to contribute.

**Comment by samjabrahams** [comment]
Will do!

**Comment by zhangpengGenedock** [comment]
@samjabrahams Hi, have you completed the "tour"? If so, where have you put it?
